### PR TITLE
Use stable branch for gz-cmake4

### PR DESCRIPTION
**Pull request by azeey**
Part of https://github.com/gazebo-tooling/release-tools/issues/1092

### CODE REVIEW

**Comment by scpeters** [comment]
I think we need to change the `gz-cmake` branch for every single Ionic package file and the collection file, right?

**Comment by azeey** [comment]
> I think we need to change the `gz-cmake` branch for every single Ionic package file and the collection file, right?

Done in https://github.com/gazebo-tooling/gazebodistro/pull/165/commits/4d842af500e000767e2883c556775c3caf46fc0c

File: collection-ionic.yaml [hunk 1]
```diff
@@ -1,9 +1,8 @@
----
```
Comment (scpeters) [inline collection-ionic.yaml:1]: the yaml linter is warning about this first line of `---` being deleted, though the workflow is still passing

Comment (azeey) [reply collection-ionic.yaml]: Fixed in https://github.com/gazebo-tooling/gazebodistro/pull/165/commits/a979b044560e89fb6dfc5d9a471bf5a0bff5e09c

**Review by scpeters — APPROVED** [review]

**Pull request merged**